### PR TITLE
Improve docs for the active property in access rules

### PR DESCRIPTION
**Pull request by nwalters512**
This is based on feedback that this documentation wasn't as clear as it could be. Specifically, there was confusion regarding whether or not this controlled visibility of the assessment on the assessments page. I hope these new docs are a little clearer.

### CODE REVIEW

File: docs/accessControl.md [hunk 1]
```diff
@@ -206,11 +206,15 @@ The `showClosedAssessmentScore` access rule restriction is often used in conjunc
 
 ## Active assessments
 
-The `active` access rule restriction is useful for allowing students to see what assessments they have coming up. It should also be used when [returning exams to students](faq.md#how-do-i-give-students-access-to-view-their-exams-after-they-are-over). If `active` is set to `false`, students can see the assessment on the Assessments page, but they cannot start the assessment, create a new assessment instance, or submit answers to questions. If an assessment is currently not active but will be in the future, students can see when the assessment will become active by looking at the `Available credit` column on the Assessments page. The `active` property in an access rule is `true` by default.
+The `active` access rule restriction is useful for allowing students to see what assessments they have coming up. It should also be used when [returning exams to students](faq.md#how-do-i-give-students-access-to-view-their-exams-after-they-are-over).
+
+By default, the `active` property in an access rule is `true`. This means students can start the assessment, create new assessment instances, and submit answers to questions.
```
Comment (jonatanschroeder) [inline docs/accessControl.md:211]:
```suggestion
By default, the `active` property in an access rule is `true`. This means that, if there is an access rule that applies at a particular time/context, students can start the assessment, create new assessment instances, and submit answers to questions.
```

File: docs/accessControl.md [hunk 2]
```diff
@@ -206,11 +206,15 @@ The `showClosedAssessmentScore` access rule restriction is often used in conjunc
...
 
 If `active` is set to `false` in an access rule, the available `credit` cannot be set to any value other than 0 (the default value).
 
-Note that when `"active": false` is used to make assessments visible to students after they've finished the assessment, students can still access and modify files in any [workspaces](workspaces/index.md) associated with questions on the assessment. However, they will not be able to submit any changed workspace files for grading.
+Note that when `"active": false` is used to allow student to review their work after they've finished the assessment, students can still access and modify files in any [workspaces](workspaces/index.md) associated with questions on the assessment. However, they will not be able to submit any changed workspace files for grading.
```
Comment (jonatanschroeder) [inline docs/accessControl.md:217]:
```suggestion
Note that when `"active": false` is used to allow students to review their work after they've finished the assessment, students can still access and modify files in any [workspaces](workspaces/index.md) associated with questions on the assessment. However, they will not be able to submit any changed workspace files for grading.
```

**Review by jonatanschroeder — COMMENTED** [review]
I think it's worth clarifying that, if there is no access rule that applies in a context, that the assessment will not be visible, and students will not be able to see anything other than their score (if they have already started it).

**Comment by jonatanschroeder** [comment]
> I think it's worth clarifying that, if there is no access rule that applies in a context, that the assessment will not be visible, and students will not be able to see anything other than their score (if they have already started it).

To be clear, I don't think this was not addressed in my suggestions, do you believe this should be added? In other words, an explicit explanation about what the rules are (in terms of what is visible and/or permitted) when no access rule is currently in place?

**Comment by nwalters512** [comment]
Do you think that specifically belongs in the documentation for `active`? I'd imagine that this would be better suited as a separate section. That's a good suggestion, but if it's not specifically related to `active`, it's not relevant to this PR.

**Comment by nwalters512** [comment]
Do you think that specifically belongs in the documentation for `active`? I'd imagine that this would be better suited as a separate section. That's a good suggestion, but if it's not specifically related to `active`, it's not relevant to this PR.

**Comment by jonatanschroeder** [comment]
> Do you think that specifically belongs in the documentation for `active`? I'd imagine that this would be better suited as a separate section. That's a good suggestion, but if it's not specifically related to `active`, it's not relevant to this PR.

I think it's worth contrasting it to no rule, so instructors can decide if a new role is needed.

**Comment by nwalters512** [comment]
@jonatanschroeder I drafted this, is this what you're looking for?

> If there is no applicable access rule at a particular time/context, students will not be able to view the assessment, and it won't appear on their Assessments page. They will only be able to see their grade for the assessment in the gradebook.

Otherwise, please provide me with the text you'd like to see and I can incorporate it into this PR. You're also more than welcome to open a PR of your own.

**Comment by jonatanschroeder** [comment]
> @jonatanschroeder I drafted this, is this what you're looking for?
> 
> > If there is no applicable access rule at a particular time/context, students will not be able to view the assessment, and it won't appear on their Assessments page. They will only be able to see their grade for the assessment in the gradebook.
> 
> Otherwise, please provide me with the text you'd like to see and I can incorporate it into this PR. You're also more than welcome to open a PR of your own.

That sounds good.

**Comment by nwalters512** [comment]
@jonatanschroeder added in 5d6359a. I wasn't sure exactly where this should go re: the order of paragraphs; I'm open to suggestions of moving it up or down.

**Review by jonatanschroeder — APPROVED** [review]

**Pull request merged**